src/pokete/classes/landscape.py

```
@@ -43,22 +44,38 @@ def action(self, ob):
             or (not n_a and not is_night)
             or (n_a and is_night)
         }
-        if random.randint(0, 8) == 0:
+
+        # Encounter rate adjustments: PRESSURE = higher, RELIEF = lower
+        encounter_chance = 8
+        pacing = dm.get_pacing_mode()
+        if pacing == "PRESSURE": encounter_chance = 6
```

Ruff (0.14.14)

Multiple statements on one line (colon)

(E701)